src/pyOpenMS/tests/unittests/test_psm_dataframe.py

```
@@ -0,0 +1,832 @@
+    pep_id2 = oms.PeptideIdentification()
+    pep_id2.setHigherScoreBetter(True)
+    hit2 = oms.PeptideHit()
+    hit2.setSequence(oms.AASequence.fromString("TESTPEP"))
+    hit2.setCharge(2)
+    pep_id2.setHits([hit2])
+    pep_ids.append(pep_id2)
+
+    df = pep_ids.get_psm_df()
+
+    assert "higher_score_better" in df.columns
+    assert df.iloc[0]["higher_score_better"] == False
```

Ruff (0.14.10)

Avoid equality comparisons to `False`; use `not df.iloc[0]["higher_score_better"]:` for false checks

Replace with `not df.iloc[0]["higher_score_better"]`

(E712)